src/js/configureStore.js

```diff
@@ -2,8 +2,12 @@ import { applyMiddleware, compose, createStore } from 'redux';
 import logger from 'redux-logger';
 import { routerMiddleware } from 'react-router-redux';
 import thunk from 'redux-thunk';
+import reduxLocalStorage from 'redux-localstorage';
 import reducer from './reducers';
 
+const persistState = reduxLocalStorage(undefined, {
```

Comment:
If you want to select just part of the state, you can pass a string or an array specifying.